packages/app-project/test/stubPanoptesJs/stubPanoptesJs.js

```diff
@@ -0,0 +1,16 @@
+import projectFixture from '../fixtures/project'
+
+const stubPanoptesJs = {
```

Comment:
We could consider adding a helper function to the panoptes.js project resource request set to request by project slug and have the tests for it there. This is a pretty comment kind of request query.

Reply:
cc https://github.com/zooniverse/front-end-monorepo/issues/34